network/src/peer_store/peer_store_db.rs

```diff
@@ -49,25 +49,19 @@ impl BanList {
 }
 
 impl PeerStore {
-    pub fn load_from_dir<P: AsRef<Path>>(path: P) -> Result<Self, Error> {
+    pub fn load_from_dir<P: AsRef<Path>>(path: P) -> Self {
```

Comment:
I prefer a new method `load_from_dir_or_default`. It just not feels right to silently discard the error.

Reply:
renamed method accordingly and added the error log when failed to load data from file (empty or broken), test case is added also, please review again, thanks